### PR TITLE
Dismemberment wounds

### PR DESCRIPTION
**Pull request by BobJoga**
## About The Pull Request

Dismembering a limb causes a wound equivalent to a weeping avulsion on the chest.

## Why It's Good For The Game

Dismemberment should be lethal, not something preferrable to having a severely damaged limb - This makes dismemberment scary and very likely to kill you if left untreated.

## Changelog
:cl:
balance: When a limb gets dismembered, it now causes a critical slash wound on the chest.
/:cl:

### CODE REVIEW

**Comment by itseasytosee** [comment]
powercreep for nugget smite

**Comment by Timberpoes** [comment]
How, if at all, does this interact with the genetics mutation autonomy?

```
Autonomy
Allows a creature to voluntarily discard a random limb, to help it escape from predators. 
```

**Comment by BobJoga** [comment]
> How, if at all, does this interact with the genetics mutation autonomy?
> 
> ```
> Autonomy
> Allows a creature to voluntarily discard a random limb, to help it escape from predators. 
> ```

If the limb is simply dropped, it does not apply the wound.

**Comment by Ryll-Ryll** [comment]
My only concern is that it's already pretty easy to lose lots of blood, while this is definitely something that would make sense realistically, I'm wary about continuing to stack negative effects on someone who already just lost their damn arm

**Comment by BobJoga** [comment]
> My only concern is that it's already pretty easy to lose lots of blood, while this is definitely something that would make sense realistically, I'm wary about continuing to stack negative effects on someone who already just lost their damn arm

I've downplayed the immediate blood loss of losing a limb exactly because of that.

**Comment by JohnFulpWillard** [comment]
I thought dismembering a limb was the ghetto version to temporarily deal with wounds on said limb, this just kinda goes against that

**Comment by wesoda25** [comment]
> I thought dismembering a limb was the ghetto version to temporarily deal with wounds on said limb, this just kinda goes against that

Kinda the point of the PR

**Comment by Timberpoes** [comment]
> 
> 
> My only concern is that it's already pretty easy to lose lots of blood, while this is definitely something that would make sense realistically, I'm wary about continuing to stack negative effects on someone who already just lost their damn arm

Funnily enough, a powergamer move is to get the surgery that lets you reattach limbs but has them fall off easier. If you get hit by an explosion of some sort, a bunch of your limbs fly off and you can actually avoid entering crit compared to a person who was stood next to you and has all their limbs still attached.

**Comment by optimumtact** [comment]
Sure you're alive, but now you're a nugget, so was it worth it

**Comment by Timberpoes** [comment]
> 
> 
> Sure you're alive, but now you're a nugget, so was it worth it

It is only worth it if you say so in chat.

**Comment by Tlaltecuhtli** [comment]
does using the amputator give you free wounds?

File: code/datums/wounds/loss.dm [hunk 1]
```diff
@@ -46,9 +48,21 @@
...
+	var/obj/item/bodypart/chest/chest = victim.get_bodypart(BODY_ZONE_CHEST)
+	if((dismembered_part.body_zone != BODY_ZONE_CHEST) && chest)
+		apply_wound(chest, TRUE)
+	else
+		qdel(src)
+
+/datum/wound/slash/critical/loss/get_examine_description(mob/user)
+	. = ..()
+	if(fake_body_zone == BODY_ZONE_HEAD)
+		return "<span class='deadsay'>[..()]</span>"
```
Comment (Rohesie) [inline code/datums/wounds/loss.dm:68]:
```suggestion
	. = ..()
	if(fake_body_zone == BODY_ZONE_HEAD)
		return "<span class='deadsay'>[.]</span>"
```
Don't call parent twice.

File: code/datums/wounds/_wounds.dm [hunk 2]
```diff
@@ -42,6 +42,8 @@
 	var/mob/living/carbon/victim = null
 	/// The bodypart we're parented to
 	var/obj/item/bodypart/limb = null
+	/// A phantom body zone, for situations where the wound is tied to a bodypart that doesn't exist on the victim, like dismemberment
+	var/fake_body_zone
```
Comment (Rohesie) [inline code/datums/wounds/_wounds.dm:46]:
```suggestion
	var/phantom_body_zone = null
```
You can really not describe it any better than this. Can you update the variable name to match the very fitting description?

File: code/modules/surgery/bodyparts/dismemberment.dm [hunk 3]
```diff
@@ -368,6 +368,12 @@
 		var/datum/wound/W = i
 		W.apply_wound(src, TRUE)
 
+	/// Reattaching a limb fixes dismemberment wounds
+	var/obj/item/bodypart/chest/chest = C.get_bodypart(BODY_ZONE_CHEST)
+	for(var/datum/wound/woundie in chest.wounds)
```
Comment (Rohesie) [inline code/modules/surgery/bodyparts/dismemberment.dm:373]:
```suggestion
	for(var/w in chest.wounds)
		var/datum/wound/woundie = w
```
No need to type-check, correct?

**Comment by Rohesie** [comment]
@Ryll-Ryll , you are the author of the wounds system. Could you request concrete changes, approve or deny? It's your feature, and you should know its balance better than anyone here.

**Comment by Winterous** [comment]
While realistic this definitely raises balance concerns, weeping avulsion is incredibly lethal, and this may well just turn plasma cutters into horrible murderboning weapons due to how easy it is to sever limbs with them. Perhaps plasma cutters should like, self-cauterize the wound? I'm not certain how they're supposed to work to be honest.

Also, will the wound be applied if you sever a robotic limb? If so, it probably shouldn't, or cause a lesser wound at least.
On that note, what if your TORSO is robotic? Can a robotic limb even get bleeding wounds?

**Comment by BobJoga** [comment]
> While realistic this definitely raises balance concerns, weeping avulsion is incredibly lethal, and this may well just turn plasma cutters into horrible murderboning weapons due to how easy it is to sever limbs with them. Perhaps plasma cutters should like, self-cauterize the wound? I'm not certain how they're supposed to work to be honest.
> 
> Also, will the wound be applied if you sever a robotic limb? If so, it probably shouldn't, or cause a lesser wound at least.
> On that note, what if your TORSO is robotic? Can a robotic limb even get bleeding wounds?

Robotic limbs don't get wounds, so simply put this won't ever work on them. That said, i feel like plasma cutters causing dismemberment is itself the balance concern rather than anything else.

**Comment by Winterous** [comment]
No i mean, if you have an organic torso, but a robotic arm, and the arm gets blown off; will you suffer a wound to your torso from that?
Because in theory it's not attached to blood supply, so shouldn't cause a major bleed.

**Comment by Ryll-Ryll** [comment]
There are lots of assumptions that the person getting dismembered has flesh and blood that need to be addressed. 

In addition, even with the decrease in initial bloodloss from dismemberment, I still feel like getting a full avulsion for every limb lost is far too punishing for someone who already lost a limb to begin with. I've already been walking back how quickly people lose blood and adding in new ways to stem bloodloss with my recent PRs, this pushes too far in the other direction I think. Yeah it's a bit silly that you may be better off losing a limb in certain cases, but you're still losing a significant amount of functionality even then

**Pull request closed without merging**